README.md

```diff
@@ -26,20 +26,22 @@ The following configuration options are optional.
 | CALLER_KEY            | The caller key that WiseTime should provide with post time webhook calls. The connector does not authenticate Webhook calls if not set. |
 | TAG_UPSERT_PATH       | The WiseTime tag folder path to use for Jira tags. Defaults to `/Jira/` (trailing slash is required). Use `/` for root folder.          |
 | TAG_UPSERT_BATCH_SIZE | Number of tags to upsert at a time. A large batch size mitigates API call latency. Defaults to 500.                                     |
-| PROJECT_KEYS_FILTER   | If set, the connector will only handle Jira issues from the configured Jira project keys.                                               | 
+| PROJECT_KEYS_FILTER   | If set, the connector will only handle Jira issues from the configured Jira project keys.                                               |
+| PERSISTENT_DIR        | If set, the connector will use the directory as the storage location for SQLite to keep track the Jira issues it have synced.           |
```

Comment:
What happens if not set? Is there a default path that the connector will use?

Reply:
Yes. the connector library will generate a directory in `/tmp`:
https://github.com/wisetime-io/wisetime-connector-java/blob/master/src/main/java/io/wisetime/connector/datastore/FileStore.java#L97-L109

Since we need to use the volume, we should specify the `PERSISTENT_DIR` so it will always look on the desired DB file everytime connector restart

Reply:
Okay, can you document the default behaviour here as well?